pkg/tsdb/elasticsearch/error_handling_test.go

```diff
@@ -43,7 +43,7 @@ func TestErrorAvgMissingField(t *testing.T) {
 	require.NoError(t, err)
 
 	// FIXME: we should return the received error message
-	require.Len(t, result.response.Responses, 0)
+	require.Equal(t, "unexpected status code: 400", result.response.Responses["A"].Error.Error())
```

Comment:
Previously, the middleware returned error as an error, see https://github.com/grafana/grafana-plugin-sdk-go/blob/main/experimental/errorsource/error_source_middleware.go#L26. We are now moving processing to Elasticsearch and returning it as error response.